docs/example_applications_algorithms.rst

```diff
@@ -70,7 +71,8 @@ The following tutorials and quickstart guides walk you through some of these exa
 
   8. **Experiment tracking**
 
-    * :ref:`TensorBoard Streaming <tensorboard_streaming>` - Example building on Hello PyTorch with TensorBoard streaming from clients to server
+    * :ref:`FL Experiment Tracking with TensorBoard Streaming <tensorboard_streaming>` - Example building on Hello PyTorch with TensorBoard streaming from clients to server
+    * :ref:`FL Experiment Tracking with MLflow <experiment_tracking_mlflow>` - Example integrating Hello PyTorch with MLflow with streaming from clients to server
```

Comment:
The ref is different from what you have in docs/examples/fl_experiment_tracking_mlflow.rst
(experiment_tracking_mlflow VS _experiment_tracking:)

Reply:
fixed